python/GafferSceneUI/_SceneViewInspector.py

```diff
@@ -867,6 +875,19 @@ def popup( self, position ) :
 			else :
 				textWidget.setFocussed( True )
 
+	def __visibilityChaged( self, unused ) :
+
+		if self.visible() :
+			return
+
+		# The modal popup will hide the window without de-focusing the
+		# current widget. Ultimately this results in un-commited edits
```

Comment:
commited -> committed

Reply:
I should probably be committed really, for the sake of everyone who has to read all these novel combinations of letters, [see](https://github.com/GafferHQ/gaffer/pull/3974/commits/6a16d4dd56da3058663e108bf8251a7185068fba).